packages/studio-base/src/panels/ThreeDeeRender/renderables/OccupancyGrids.ts

```diff
@@ -28,41 +28,44 @@ import {
 import { ColorRGBA, OccupancyGrid, OCCUPANCY_GRID_DATATYPES } from "../ros";
 import { BaseSettings } from "../settings";
 import { topicIsConvertibleToSchema } from "../topicIsConvertibleToSchema";
+import PNG from "png-ts";
+
 
 type ColorModes = "custom" | "costmap" | "map" | "raw";
 
 export type LayerSettingsOccupancyGrid = BaseSettings & {
   frameLocked: boolean;
-  minColor: string;
-  maxColor: string;
-  unknownColor: string;
-  invalidColor: string;
-  colorMode: ColorModes;
-  alpha: number;
+  // minColor: string;
+  // maxColor: string;
+  // unknownColor: string;
+  // invalidColor: string;
+  // TODO: These were not on original commit, commenting them out anyways
+  // colorMode: ColorModes;
+  // alpha: number;
 };
 
 const INVALID_OCCUPANCY_GRID = "INVALID_OCCUPANCY_GRID";
 
-const DEFAULT_MIN_COLOR = { r: 1, g: 1, b: 1, a: 1 }; // white
-const DEFAULT_MAX_COLOR = { r: 0, g: 0, b: 0, a: 1 }; // black
-const DEFAULT_UNKNOWN_COLOR = { r: 0.5, g: 0.5, b: 0.5, a: 1 }; // gray
-const DEFAULT_INVALID_COLOR = { r: 1, g: 0, b: 1, a: 1 }; // magenta
-const DEFAULT_ALPHA = 1.0;
+// const DEFAULT_MIN_COLOR = { r: 1, g: 1, b: 1, a: 1 }; // white
+// const DEFAULT_MAX_COLOR = { r: 0, g: 0, b: 0, a: 1 }; // black
+// const DEFAULT_UNKNOWN_COLOR = { r: 0.5, g: 0.5, b: 0.5, a: 1 }; // gray
+// const DEFAULT_INVALID_COLOR = { r: 1, g: 0, b: 1, a: 1 }; // magenta
+// const DEFAULT_ALPHA = 1.0;
```

Comment:
There's a lot of code commented out in this file, is this intentional?

Reply:
Yeah this was all apart of the original MiR edited code.
I can remove it but was thinking we might want it. Your call

Reply:
The modified files outside the mir folder were breaking the docker build and I thought they were leftovers from the migration so I reset them. Sounds like it was a mistake.

Are we keeping any modifications outside of the mir folder?